subsys/logging/log_cmds.c

```diff
@@ -363,6 +363,42 @@ static int cmd_log_backends_list(const struct shell *shell,
 	return 0;
 }
 
+static int cmd_log_strdup_utilization(const struct shell *shell,
```

Comment:
Could we add #ifdef CONFIG_LOG_STRDUP_POOL_PROFILING around this function?

Reply:
See new shell feature:
`SHELL_COND_CMD_ARG(CONFIG_LOG_STRDUP_POOL_PROFILING, ...)`
Command and function is not compiled when CONFIG_LOG_STRDUP_POOL_PROFILING is not enabled.

Reply:
Ok